templates/docker-compose.yaml.j2

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+services:
+    # web:
+    #   image: nginx:latest
+    #   ports:
+    #     - "8080:80"
+    #   volumes:
+    #     - /opt/nginx/html:/usr/share/nginx/html:ro
+    #     - /opt/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+  cache:
+    image: nginx:latest
+    ports:
+      - "127.0.0.1:80:80"
+    volumes:
+      # - /opt/nginx/cache-test:/var/cache/nginx
```

Comment:
should this be commented?i think currently the cache willlive in the docker container which is fine with me

Reply:
let's just drop the line if anything?